src/interfaces/alteration-interfaces/resolutions-interface.ts

```diff
@@ -2,5 +2,5 @@
 export interface ResolutionsIF {
   date: string,
   id: number,
-  type: string
+  type: string,
```

Comment:
Comma is not needed at end.

Actually, commas aren't needed for any property in an interface.